packages/terra-grid/CHANGELOG.md

```diff
@@ -0,0 +1,71 @@
+Terra Grid Changelog
+====================
+
+3.1.0 - 2017-06-20
+------------------
+### Enhancements
```

Comment:
I'm OK with current CHANGLOG. For dependency, consumers can view the`package.json` directly to understand the required packages for the component. That's also easy to maintain.